easybuild/easyconfigs/j/JAGS/JAGS-4.3.0-foss-2019a.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'ConfigureMake'
+
+name = 'JAGS'
+version = '4.3.0'
+
+homepage = 'http://mcmc-jags.sourceforge.net/'
+description = """JAGS is Just Another Gibbs Sampler.  It is a program for analysis 
+ of Bayesian hierarchical models using Markov Chain Monte Carlo (MCMC) simulation  """
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = [('http://sourceforge.net/projects/mcmc-jags/files/JAGS/%(version_major)s.x/Source/', 'download')]
+sources = [SOURCE_TAR_GZ]
+checksums = ['8ac5dd57982bfd7d5f0ee384499d62f3e0bb35b5f1660feb368545f1186371fc']
+
+configopts = ' --with-blas="-lopenblas" --with-lapack="-lopenblas" '
```

Comment:
why hardcoding `-lopenblas` here? using `$LIBBLAS` and `$LIBLAPACK` is in general better, for instance to allow using `--try-toolchain`

Reply:
good suggestion, I had no idea those variables existed. Fixed :+1: